scripts/run-prow-test.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# This script runs e2e tests on the AWS Load Balancer Controller
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GINKGO_TEST_BUILD="$SCRIPT_DIR/../test/build"
+
+source "$SCRIPT_DIR"/lib/common.sh
+
+function toggle_windows_scheduling(){
+  schedule=$1
+  nodes=$(kubectl get nodes -l kubernetes.io/os=windows | tail -n +2 | cut -d' ' -f1)
+  for n in $nodes
+  do
+    kubectl $schedule $n
+  done
+}
+
+echo "Cordon off windows nodes"
+toggle_windows_scheduling "cordon"
+
+eksctl utils associate-iam-oidc-provider \
+    --region $REGION \
+    --cluster $CLUSTER_NAME \
+    --approve
+
+echo "Creating AWSLoadbalancerController IAM Policy"
+curl -o iam-policy.json https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.2.1/docs/install/iam_policy.json
```

Comment:
i think better to use "docs/install/iam_policy.json" in the working directory, so that we are testing the policy as well.